test/test_eos_tabulated.cpp

```diff
@@ -261,54 +259,6 @@ SCENARIO("SpinerEOS depends on rho and sie", "[SpinerEOS],[DependsRhoSie]") {
     steelEOS.Finalize();      // cleans up device memory
   }
 }
-
-SCENARIO("EOS Builder and SpinerEOS",
```

Comment:
Doesn't make sense to test this anymore, as there's no special code path for the builder. Testing modifiers elsewhere does the trick.